src/ansys/mapdl/core/mapdl_grpc.py

```diff
@@ -2767,6 +2768,26 @@ def xpl(self) -> "ansXpl":
             self._xpl = ansXpl(self)
         return self._xpl
 
+    @property
+    def plugin(self) -> "ansPlugin":
+        """MAPDL plugin handler
+
+        Plugin Manager for MAPDL
+
+        Examples
+        --------
+
+        >>> from ansys import Mapdl
+        >>> mapdl = Mapdl()
+        >>> plugin = mapdl.plugin
+        >>> plugin.load('PluginDPF')
+        """
+        if self._plugin is None:
+            from ansys.mapdl.core.plugin import ansPlugin
```

Comment:
```suggestion
```

It seems to me there is no reason to delay the import. You can delete this line and then apply my previous comment (remove quotes). 

Or delete the `import` at the beginning and leave these lines as it is.